distributed/deploy/tests/test_subprocess.py

```diff
@@ -72,6 +75,24 @@ async def test_raise_if_scheduler_fails_to_start():
             pass
 
 
+@pytest.mark.skipif(WINDOWS, reason="distributed#7434")
+@gen_test()
+async def test_subprocess_cluster_does_not_depend_on_logging():
+    async def _start():
+        async with SubprocessCluster(
+            asynchronous=True,
+            dashboard_address=":0",
+            scheduler_kwargs={"idle_timeout": "5s"},
+            worker_kwargs={"death_timeout": "5s"},
+        ):
+            pass
+
+    with new_config_file(
+        {"distributed": {"logging": {"distributed": logging.CRITICAL + 1}}}
+    ):
+        await asyncio.wait_for(_start(), timeout=2)
```

Comment:
I don't understand what's the purpose of this super-short timeout, and why the timeout integrated in gen_test is not what we want in this case?

Reply:
I write the test before the fix and didn't want to wait the entire 30 seconds while fixing.